packages/prices-api/src/llamma/api.ts

```diff
@@ -29,14 +61,34 @@ export async function getTrades(endpoint: Endpoint, chain: Chain, llamma: string
   }
 }
 
-export async function getOHLC(endpoint: Endpoint, chain: Chain, llamma: string, options?: Options) {
+type GetOHLCParams = {
+  endpoint: Endpoint
+  chain: Chain
+  llamma: Address
+  interval?: number
+  units?: 'day' | 'hour' | 'minute'
+  start?: number
+  end?: number
+}
+
+export async function getOHLC(
+  { endpoint, chain, llamma, interval = 1, units = 'hour', start, end }: GetOHLCParams,
+  options?: Options,
+) {
   const host = await getHost(options)
 
-  const end = Math.floor(new Date().getTime() / 1000)
-  const start = end - 10 * 24 * 60 * 60 // Subtract 1 month worth of seconds.
+  end ??= Math.floor(new Date().getTime() / 1000)
+  start ??= end - 10 * 24 * 60 * 60 // Subtract 1 month worth of seconds.
```

Comment:
this doesn't belong in this function imo, shouldn't it be shared between other calls?

Reply:
I've created a `getTimeRange` util function specific to the prices API package for now. I've also refactored existing api calls that used ranges to use that function. Its inputs are unix timestamps for now as that matches best with the optional start and end parameters of the function calls that are in unix timestamps already. You'll notice that for a few  `end - 10 * 24 * 60 * 60 // Subtract 1 month worth of seconds.` the comment was wrong. It should've been 10 days in the first place instead of a month. These defaults can be changed later when deemed necessary.